packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -162,7 +163,11 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
         retry = true;
       }
 
-      if (retry && this.webex.internal.mercury.connected) {
+      if (
+        retry &&
+        this.webex.internal.mercury.connected &&
+        line.status !== RegistrationStatus.IDLE
+      ) {
         retry = await line.registration.handleConnectionRestoration(retry);
```

Comment:
One comment for the log inside handleConnectionRestoration. Can we change it to "Mercury connection is up again, re-registering with Webex Calling if needed."

Reply:
Updated